charts/admission-controller/templates/_helpers.tpl

```diff
@@ -411,7 +411,9 @@ webhooks:
       name: {{ include "admissionController.webhook.fullname" . }}
       path: /validate
       port:  {{ .Values.webhook.v2.service.port }}
-    caBundle: {{ .Cert }}
+    {{- with .Cert }}
```

Comment:
This was the laziest way I could think of to avoid reimplementing this template.